packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.4.0",
+    "@okta/courage": "^4.6.0-beta.2093.ga073567",
```

Comment:
we should keep this as an specific version (the `^` means it is that version or above, s we don't need that)